packages/client/src/comp/states/Kansas.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import Select from 'muicss/lib/react/select'
+import { BaseInput } from '../util/Input'
+
+import { KansasInfo } from '../../common'
+import { SignatureBase, StatelessInfo, NoSignature } from './Base'
+import { Upload } from '../util/Upload'
+import { useControlRef } from '../util/ControlRef'
+import { TogglableDropdown } from '../util/TogglableDropdown'
+
+const options = {
```

Comment:
I usually just make it the actual string I'm going to show on the website (i.e. an array is sufficient).